skt/state_file.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python2
+
+# Copyright (c) 2017 Red Hat, Inc. All rights reserved. This copyrighted
+# material is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General
+# Public License v.2 or later.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Functions that manage the skt state file."""
+import logging
+import os
+import yaml
+
+
+def destroy(cfg):
+    """
+    Destroy the state file.
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+
+    if os.path.isfile(state_file):
+        try:
+            os.unlink(state_file)
+        except IOError as exception:
+            logging.error("Failed to update state file: %s", exception)
```

Comment:
I'm so sorry to bother you again in this pull, but shouldn't this be "Failed to *remove* state file"?

Reply:
Right. I'll fix it.